File: LorePdks.COMMON/DTO/Firma/FirmaDTO.cs
using LorePdks.COMMON.DTO.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LorePdks.COMMON.DTO.Firma
{


    public class FirmaDTO : BaseDTO
    {
        public string ad { get; set; }
        public string kod { get; set; }
        public string aciklama { get; set; }
        public string adres { get; set; }
        public decimal? mesaiSaat { get; set; }
        public decimal? molaSaat { get; set; }        public decimal? cumartesiMesaiSaat { get; set; }
        public decimal? cumartesiMolaSaat { get; set; }
        public bool isPdks { get; set; }
        public bool isAlarm { get; set; }
        public bool isKamera { get; set; }
    }
}

File: LorePdks.WEB/src/app/core/models/FirmaDto.ts
import { baseDto } from "./baseDto";

/**
 * Firma bilgisini temsil eden model
 */
export class FirmaDto extends baseDto {
  ad: string;
  kod: string;
  aciklama: string;
  adres: string;
  mesaiSaat?: number;
  molaSaat?: number;  cumartesiMesaiSaat?: number;
  cumartesiMolaSaat?: number;
  isPdks: boolean;
  isAlarm: boolean;
  isKamera: boolean;
}